melodies_monet/_cli.py

```diff
@@ -914,6 +915,221 @@ def get_utc_offset(*, lat, lon):
             ds.to_netcdf(dst / out_name)
 
 
+@app.command()
+def get_aqs(
+    start_date: str = typer.Option(..., "-s", "--start-date", help=f"Start date. {_DATE_FMT_NOTE}"),
+    end_date: str = typer.Option(..., "-e", "--end-date", help=f"End date. {_DATE_FMT_NOTE} {_DATE_END_NOTE}"),
+    daily: bool = typer.Option(False, help=(
+            "Whether to retrieve the daily averaged data product. "
+            "By default, the hourly data is fetched."
+        )
+    ),
+    param: List[str] = typer.Option(["SPEC", "PM2.5", "PM10", "VOC"], "-p", "--params", help=(
```

Comment:
@rschwant @bbakernoaa what do you think defaults should be?

Reply:
@rschwant suggested default could be O3, PM2.5, PM10

Reply:
Yes, I agree. I'm testing this out right now too.

Reply:
Updated in e54849f , with some additional help text.